src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/LanguageClient/ProjectSystem/RazorProjectInfoEndpointPublisher.cs

```diff
@@ -77,7 +77,8 @@ public void StartSending()
         }
     }
 
-    private void ProjectManager_Changed(object sender, ProjectChangeEventArgs args)
+    // internal for testing
+    internal void ProjectManager_Changed(object sender, ProjectChangeEventArgs args)
```

Comment:
Please use the test accessor pattern rather than marking methods as "internal for testing".

Reply:
Done